Jenkinsfile

```diff
@@ -575,15 +651,11 @@ pipeline {
                                 '$BUILDARGS_QB_CHECK' +
                                 ' --build-arg QUICKBUILD_DEPS="' +
                                   env.QUICKBUILD_DEPS_EL7 + '"' +
-                                ' --build-arg REPOS="' + component_repos() + '"'
+                                ' --build-arg REPOS="' + pr_repos() + '"'
```

Comment:
Ditto. You can assume _ditto_ for all further instances of the use of `pr_repos()`.  I won't point each one out.